eZ/Publish/API/Repository/Values/User/Role.php

```diff
@@ -38,6 +55,41 @@
     protected $identifier;
 
     /**
+     * The status of the role.
+     *
+     * @var int One of Role::STATUS_DEFINED|Role::STATUS_DRAFT|Role::STATUS_MODIFIED
+     */
+    protected $status;
```

Comment:
shouldn't this be on the RoleDraft object?
Role object implicit means it is published.